selfapi.js

```diff
@@ -121,12 +121,29 @@ API.prototype = {
   },
 
   // Export API documentation as HTML.
-  toHTML: function (basePath) {
+  toHTML: function (basePath, anchors) {
```

Comment:
`anchors` is an array of unique ID attributes used in the HTML code.

The argument can safely be omitted, however it should be passed down to API children in order to guarantee unique anchors throughout the whole HTML code (I may have forgotten to pass it down, so I'll fix that).

Also, it will allow us to add a Table of Contents with helpful links to the generated HTML.